deploy/helm/sumologic/conf/logs/collector/otelcol/config.yaml

```diff
@@ -26,7 +26,7 @@ processors:
     send_batch_size: 1000
     timeout: 1s
 
-{{- if .Values.sumologic.logs.systemd.enabled }}
+{{- if or .Values.sumologic.logs.systemd.enabled .Values.sumologic.logs.kubelet.enabled }}
```

Comment:
just curious, 

what is the reason to split kubelet logs from systemd?

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/3168
This PR has the history @chan-tim-sumo , This was added in OSC-KP Epic and seems dominik was the one previously working on it. 
Scenario is like a customer would want only kubelet logs and not other systemd logs, Now, we can easily toggle kubelet and systemd logs with a flag in values yaml.